spyder_kernels/console/shell.py

```diff
@@ -62,6 +65,18 @@ def __init__(self, *args, **kwargs):
         # register post_execute
         self.events.register('post_execute', self.do_post_execute)
 
+    def init_magics(self):
```

Comment:
Add the runfile and other functions to the builtins (they will be turned into magics later)